README.adoc

```diff
@@ -59,118 +62,91 @@ include::{common-includes}/gitclone.adoc[]
 // =================================================================================================
 // Static guide instruction
 ifndef::cloud-hosted[]
+== Starting and preparing your cluster for deployment
+
+Start your {kube} cluster.
+
+Run the following command from a command-line session:
+
+[role=command]
+```
+minikube start
+```
+
+If you run Minikube by root user, you can append the `--force` option to the above command.
+
+Next, validate that you have a healthy {kube} environment by running the following command from the active command-line session.
```

Comment:
```suggestion
Next, validate that you have a healthy `{kube}` environment by running the following command from the active command-line session.
```

Reply:
{kube} will convert to Kubernetes, i.e. "...a healthy Kubernetes environment..."
 
 if add \`s, it will be   "...a healthy `Kubernetes` environment..."